web/modules/custom/portland/portland.module

```diff
@@ -31,7 +31,9 @@ const GROUP_CONTENT_TYPE_PLURAL = [
     "page" => ["pages"],
     "development_permit" => ["permits"],
     "location" => ["locations"],
-    "calendar_item" => ["calendar/archive"]
+    "calendar_item" => ["calendar/archive"],
+    "city_code" => ["code"],
```

Comment:
@jookyg, I'm surprised to see this change in this branch as well as the code/charter branch. Was one branched from the other? Regardless, we don't need this pluralization in either branch.

Reply:
Resolved